frontends/slack/src/main/kotlin/com/gatehill/corebot/chat/SlackSessionServiceImpl.kt

```diff
@@ -42,8 +43,18 @@ open class SlackSessionServiceImpl @Inject constructor(configService: ConfigServ
     override val botUsername: String
         get() = session.sessionPersona().userName
 
-    override fun sendMessage(channelId: String, message: String) {
-        session.sendMessage(session.findChannelById(channelId), message)
+    override fun sendMessage(channelId: String, threadTimestamp: String, message: String) {
+        if(threadTimestamp == ""){
```

Comment:
Rather than have the check for 'reply in thread' in lots of places, could we just do the check here once, and always pass in the thread/event timestamp.